src/librustdoc/lib.rs

```diff
@@ -856,50 +856,41 @@ fn main_args(
             return;
         }
 
-        compiler.enter(|queries| {
-            let mut gcx = queries.global_ctxt();
-            if sess.dcx().has_errors().is_some() {
-                sess.dcx().fatal("Compilation failed, aborting rustdoc");
+        let krate = rustc_interface::passes::parse(sess);
+        if sess.dcx().has_errors().is_some() {
+            sess.dcx().fatal("Compilation failed, aborting rustdoc");
+        }
```

Comment:
Looks like this accidentally got moved out of the `create_and_enter_global_ctxt()`.

Reply:
Would this be related to the rustdoc regression?